graphQL.js

```diff
@@ -1,84 +1,108 @@
-const { graphql, buildSchema } = require('graphql');
-
+const { graphql } = require('graphql');
+const { makeExecutableSchema } = require('graphql-tools');
 const db = require('./db');
 
-let plantData = db.query(`
-    SELECT * FROM plant_data
-    WHERE userid = '1';
-`)
-// plantData.then(console.log)
-
-let user = db.one(`
+let getUserById = (userid) => db.one(`
```

Comment:
Why is it db.one?  Is that a graphQL thing?

Reply:
no, it's postgresql thing.   Instead of returning an array of elements (or element) db.one returns first element of this array, which is useful to cut on code if your db should return only one element